npm-shrinkwrap.json

```diff
@@ -97,8 +97,7 @@
       "version": "1.3.3"
     },
     "bcrypt-pbkdf": {
-      "version": "1.0.0",
-      "optional": true
```

Comment:
TBH, not sure what they mean and why my shonkwrap decided to remove them here...